examples/simpleapp/mc_sc_workflow_example.md

```diff
@@ -66,14 +66,31 @@ Example of output is:
 Note: `withdrawalEpochLen` - expected to be in a range between 100 and 900 and is the same value as the one declared by `sc_create` command below.
 
 
-**Step 5: Generate data for certificate proof creation and verification**
+**Step 5: Generate Schnorr keypairs for certificate signers**
+
+As a next step we need schnorr keypairs, those keys will be used by certificate Signers
+
+For generating Schnorr key use command:
+`GenerateSchnorrKey {"seed":"my seed"}`
```

Comment:
be careful, the actual command name starts with lower case **g**: generateSchnorrKey. Same in other places below.
I am thinking, why not to add more context to current method.
It is true that this key is of type `schnorr`, but at least now is generated only for certificate signers.
And public part is used in the config file as `withdrawalEpochCertificate.signersPublicKeys`

I propose, to rename this command to `generateCertificateSignerKey {"seed":"my seed"}`
with an output 
```
{
  "signerPublicKey" : "..."
  "signerSecret": "..."
}